src/js/login/components/SearchHelpSignIn.jsx

```diff
@@ -27,7 +27,8 @@ class SearchHelpSignIn extends React.Component {
             this.props.onClickSearchHelpSignIn('account', !login.utilitiesMenuIsOpen.account);
           }}
           greeting={greeting}
-          isOpen={login.utilitiesMenuIsOpen.account}/>);
+          isOpen={login.utilitiesMenuIsOpen.account}
+          onUserLogout={this.props.onUserLogout}/>);
```

Comment:
`onUserLogout` should probably be added to the `propTypes` of both this (though this has no `propTypes` specified...) and `SignInProfileMenu` (`src/js/login/components/SignInProfileMenu.jsx`).